packages/teams-js/src/public/app.ts

```diff
@@ -33,6 +38,146 @@ import {
 } from './runtime';
 import { version } from './version';
 
+/**
+ * Number of milliseconds we'll give the initialization call to return before timing it out
+ */
+const initializationTimeoutInMs = 5000;
+
+const appLogger = getLogger('app');
+
+export function appInitializeHelper(apiVersionTag: string, validMessageOrigins?: string[]): Promise<void> {
```

Comment:
What other files are calling this function instead of the main `app.initialize` function?

Reply:
(I ask because we only need to move this helper out of the namespace and export it if another file internally needs to call it with a different api version)